### DISCUSSION TITLE
Wish to join project

### DISCUSSION THREAD

**Discussion by mzoran594** [Integration-Automation/AutoControlGUI#138] [General]
Hello,

I would like to join your project. I never did this before so could you please tell me if I can join and how to proceed? I have long experience in Python but for some time I don't have chance in my daily work to work with Python so this project would be great for me.

Regards,
Zoran

**Comment by JE-Chen** [comment]
These are the things you can do: open an issue, create a merge request (MR), or submit a feature request.
